gtk/src/toga_gtk/window.py

```diff
@@ -228,12 +266,10 @@ def set_content(self, widget):
 
     def get_size(self) -> Size:
         if GTK_VERSION < (4, 0, 0):  # pragma: no-cover-if-gtk4
-            width, height = self.native.get_default_size()
             size = self.native.get_size()
             return Size(size.width, size.height)
         else:  # pragma: no-cover-if-gtk3
-            width, height = self.native.get_default_size()
-            return Size(width, height)
+            return self._window_size
```

Comment:
Apologies if I'm forgetting this from a past review - but why is this change required? It seems less than ideal to have window size be a stateful property, rather than something determined at runtime.